app/controllers/content_types/modular_controller.rb

```diff
@@ -87,6 +87,120 @@ def one_column
     end
     # rubocop:enable Metrics/MethodLength
 
+    # GET /modular/two_column
+    # Maps to Content Types/Modular/2 Column in Cascade.
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    def two_column
+      # Cascade Data Models
+      @configuration_set = ConfigurationSet.two_column
+      @metadata_set = MetadataSet.page(title: 'Modular Two Column')
+      @data_definition = DataDefinitions::TwoColumn.default
+
+      # Set dynamic values.
+      if params[:masthead] == 'branded-old'
+        @data_definition.set_value(:masthead_type, 'Branded Masthead')
+      elsif params[:masthead] == 'slider-old'
+        @data_definition.set_value(:masthead_type, 'Slider')
+      elsif params[:masthead] == 'slider'
+        @data_definition.set_value(:masthead_type, 'Slider - New')
+      else
+        @data_definition.set_value(:masthead_type, 'Branded - New')
+      end
```

Comment:
Why not make a method that takes the `[:masthead]` parameter name and dynamically assigns the `:masthead_type` value to the data definition instance? I feel like that's probably feasible here and we would just have to ensure our naming is 1-1 with these files.
In fact, that would probably help with organization. Having a param `slider` not be equal to the `masthead_type: 'Slider'` is confusing to me.